app/views/admin/users/_header.html.erb

```diff
@@ -0,0 +1,61 @@
+<% if @user.access_locked? %>
+  <div class="crayons-notice crayons-notice--danger py-2 pr-2 mb-3 flex justify-between items-center">
+    <p class="flex-1 flex">
+      <%= crayons_icon_tag(:lock, title: "Access locked", class: "mr-2") %>
+      This user has locked access.
+    </p>
+    <%= link_to "Unlock access", unlock_access_admin_user_path(@user), method: :patch, class: "c-link c-link--block c-link--branded" %>
+  </div>
+<% end %>
+
+<section class="crayons-card flex gap-4 m:gap-6 mb-2 m:mb-3 p-3 s:p-4 m:p-7">
+  <img src="<%= @user.profile_image_url_for(length: 192) %>" width="96" height="96" alt="<%= @user.name %> profile picture" class="radius-full bg-base-10 shrink-0 hidden s:block">
```

Comment:
I intentionally hide the avatar for the smallest breakpoint to save some space and bytes on mobiles.